python/redmonster/application/zfind.py

```diff
@@ -50,7 +50,7 @@ def set_templates_from_inifile(self):
         
         if exists(self.inifile):
             self.option = SafeConfigParser()
-            self.option.optionxform = unicode
+            self.option.optionxform = str
```

Comment:
I would actually have to write a test in order to answer that question!  It seems unlikely that a redmonster config file would contain non-ascii characters though.